freediscovery/interpretation.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import re
+import matplotlib as mpl
+import matplotlib.cm as cm
+import numpy as np
+
+
+def explain_logistic_regression(weights, textlines):
+    """
+    Generate decorated HTML by adding <span> tags with background color to keywords depending on word's weight.
+
+    Parameters
+    ----------
+    weights: [dict] {vocabulary : weights} with words [str] as keys, weights (scores) [float] as values

```

Comment:
Let's make this (and all other docstrings) similar to
```
weights : dict
    <all the rest of the description>
```
so that sphinx would parse it properly. The spaces between `weights`, `:` and `dict` are necessary.